### DISCUSSION TITLE
Place bought items in the outposts cargo room right after purchase.

### DISCUSSION THREAD

**Discussion by link12313** [FakeFishGames/Barotrauma#5349] [Ideas and suggestions]
That room serves no purpose right now and this would be a good use for it, I would like to sort out my items while attached to shore power and not need to reload the save to do it.

**Comment by Xeavas** [comment]
Eh possibly have an option for it, since there's a high chance it will be left behind by accident.

**Reply by link12313** [reply]
Maybe if any item is left in the room when you start a mission is should give you a warning.

**Reply by Xeavas** [reply]
I'd rather they give it as an option as I don't want to manually load all cargo down the docking port every time I buy stuff.

**Comment by Solbusaur** [comment]
I think there's a few reasons why it hasn't been done yet.
It's too easy (inconvenient, but easy) to buy materials and then use the station/ship fabricator to craft them into something that sells better. You would be able to keep doing this until the shopkeep either had nothing left to sell or no money to buy your items.
It's also possible in multiplayer for someone to camp the cargo room and take the items in the crates for themselves before the captain can distribute it, although this is more of a griefing issue than a game issue.

More so, if the buyer would click "Purchase" after each item they bought, you might run into an issue where 20 crates spawn, each carrying one item because of how the Purchase button was spammed for small batches of items. A basic, low level issue example would be
- "We bought fuel and repairing items, I don't think we forgot anything"
-- "We forgot to buy ammo"
- *Captain then goes to buy ammo*
- *A half empty crate spawns with only a bit of ammo*

More extreme cases would be 10 crates each filled with just one fuel rod.

And then of course, there's the possibility you leave a crate behind and that's money wasted.
